.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+name: Python CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - "**"
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+        python-version:
+          - 3.8
+        toxenv: [django32, quality]
```

`os`, `python-version` and `toxenv` should use same notations i.e. `-` or `[]`.

---

Travis is still using `django22` so we should keep that for now. Dropping Django22 support can be done in a different PR later on.
```suggestion
        toxenv: [django22, quality]
```